Source/SIMPLib/Common/Observable.h

```diff
@@ -55,7 +55,10 @@ class SIMPLib_EXPORT Observable : public QObject
 {
     Q_OBJECT
     PYB11_CREATE_BINDINGS(Observable)
-    PYB11_METHOD(void notifyErrorMessage ARGS humanLabel ss code)
+    PYB11_METHOD(void notifyErrorMessage ARGS humanLabel msg code)
+    PYB11_METHOD(void notifyWarningMessage ARGS humanLabel msg code)
+    PYB11_METHOD(void notifyStatusMessage ARGS humanLabel msg)
+    PYB11_METHOD(void notifyProgressMessage ARGS humanLabel msg progress)
```

Comment:
These methods no longer exist.  You might also want to consider wrapping the WithPrefix methods.

Reply:
Please wrap the appropriate methods with the PYB11_* Macros if possible.